kvmagent/kvmagent/test/test_host_plugin.py

```
@@ -1,9 +1,14 @@
+# encoding: utf-8
+
 '''
 
 @author: frank
 '''
-import unittest
+import mock
+import subprocess
```

Ruff (0.12.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)